modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/enricher/JavaConfigurationDeclarationEnricher.java

```diff
@@ -66,9 +66,9 @@ public void onSource(SourceDeclaration declaration) {
   }
 
   private void enrich(BaseDeclaration declaration, WithParameters methodWrapper) {
-    final List<ExtensionParameter> configParameters = methodWrapper.getParametersAnnotatedWith(Config.class);
+    final Set<ExtensionParameter> configParameters = methodWrapper.getParametersAnnotatedWith(Config.class);
```

Comment:
Why has this changed to Set? I understand there shouldn't be repeated elements, but that's kind of implicit in the method semantics. Doing this change might lead to unpredictable behaviour when this method returns more than one element, since iteration order becomes uncertain